content/topics/Analyze/causal-inference/did/goodmanbacon.md

```diff
@@ -0,0 +1,152 @@
+
+| City/Week | 0 | 1 | 2 |
+| -------- | ------- | ------- | ------- |
+| A | Not Treated | Treated | Treated |
+| B | Not Treated | Not Treated | Treated |
+| C | Not Treated | Not Treated | Not Treated |
+
+{{% /example %}} 
+
+The weighted average of all 2x2 comparisons is problematic for two reasons: 
+
+1. Forbidden comparisons/contrasts: in our example, one of the 2x2 comparisons that form part of the final TWFE estimate is as follows: 
+
+| City/Week | 1 | 2 |
+| -------- | ------- | ------- |
+| A | Treated | Treated |
+| B | Not Treated | Treated |
+
+The group that changes its treatment status here (i.e., the treatment group) is city B, as it goes from being untreated in week 1 to being treated in week 2. City A therefore serves as the control group, even though it is actually treated in both weeks!  
+
+This would not be a problem if the treatment effect were completely constant over time, as city A would not see any changes in the outcome from week 1 to week 2 _due to the treatment_. However, this is very rarely the case: treatment effects are often dynamic – they can vary over time due to adaptation or learning effects, for example. 
+
+The implication of these forbidden comparisons is that our supposedly causal TWFE estimates are partly composed of non-causal, biased coefficients, which actually compare treatment to treatment, as opposed to treatment to control.  
+
+2. Negative weights: each comparison is assigned a weight which is proportional to group size (the number of observations within the group) and variation in treatment exposure. The larger the group size and the smaller the variation in treatment exposure, the higher the attached weight. All weights sum to one, but they can also be negative – sometimes, the comparisons at the tails of the time period distribution (in very early or very late periods) can carry negative weights, as treatment variation is typically smaller there (as most groups are not yet treated in early stages or already treated in late stages).  
+
+{{<katex>}}
+{{</katex>}}
+
+The problem with negative weights is that they can bias our final estimate by altering the direction (sign) of the effect. Let’s suppose we have three comparisons with coefficients of 0.5, 1, and 5 and weights of 0.8, 0.8, and –0.6, respectively. Our total effect is then: $0.5*0.8 + 1*0.8 + 5*(-0.6) = -1.8$. We therefore obtain a negative treatment effect estimate even though each of the individual comparisons yields a positive effect of the treatment! 
+
+## The Goodman-Bacon decomposition 
+
+The Goodman-Bacon decomposition is a diagnostic tool which allows us to see whether our TWFE estimator is reliant on forbidden comparisons or negative weights. The decomposition breaks down the estimator into three types of comparisons: 
+
+1. Treated vs. never treated: this is equivalent to the classic 2x2 DiD comparison. We are only comparing treated to untreated units. 

```

Comment:
Very minor: Good to render the bullet headers (e.g. treated vs never treated) in bold or italics